core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -377,6 +377,7 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
   <script src="{{TEMPLATE_DIR_PREFIX}}/components/RatingComputationService.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/components/background/BackgroundBannerDirective.js"></script>
 
+  <script src="{{TEMPLATE_DIR_PREFIX}}/components/summary_tile/CollectionSummaryTileDirective.js"></script>
```

Comment:
Oops silly mistake on my part, it's used in exploration_player not collection_player, I got mixed up. I'll sort that out quickly.